DateRangeBox: add code snippets and extended description #5043 (#5111)
DevExpress/devextreme-documentation: api-reference/10 UI Components/DateBoxBase/1 Configuration/dropDownOptions.md

---
---
##### shortDescription

> Configures the drop-down field which holds the [content](/api-reference/10%20UI%20Components/dxOverlay/1%20Configuration/contentTemplate.md '/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#contentTemplate').  

Reviewer: I'm not sure what you mean by "drop-down field". Would it be okay to say "drop-down window" or simply a "drop-down"? 

```suggestion
Configures the drop-down that holds the [content](/api-reference/10%20UI%20Components/dxOverlay/1%20Configuration/contentTemplate.md '/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#contentTemplate').  
```
